src/components/UrlUtilsService.js

```diff
@@ -12,7 +12,7 @@ goog.provide('ga_urlutils_service');
         // from Angular
         // https://github.com/angular/angular.js/blob/master/src/ng/directive/input.js#L3
         var URL_REGEXP =
-            /^(ftp|http|https):\/\/(\w+:{0,1}\w*@)?(\S+)(:[0-9]+)?(\/|\/([\w#!:.?+=&%@!\-\/]))?$/;
+        /^(ftp|http|https):\/\/(?:\w+(?::\w+)?@)?[^\s/]+(?::\d+)?(?:\/[\w#!:.?+=&%@\-/[\]$'()*,;~]*)?$/;
```

Comment:
Correspond to the 1.4.8 version :  https://github.com/angular/angular.js/blob/v1.4.8/src/ng/directive/input.js#L15

The more recent regex are too permissive I think.